test/gtest/ucp/test_ucp_perf.cc

```diff
@@ -57,115 +57,155 @@ const test_perf::test_spec test_ucp_perf::tests[] =
 {
   { "tag latency", "usec",
     UCX_PERF_API_UCP, UCX_PERF_CMD_TAG, UCX_PERF_TEST_TYPE_PINGPONG,
+    UCX_PERF_WAIT_MODE_LAST,
```

Comment:
UCX_PERF_WAIT_MODE_POLL
same for all others

Reply:
@yosefe 
MODE_POLL doesn't exist in libperf.h but MODE_PROGRESS exists. MODE_PROGRESS repeatedly calls progress. 
Are you suggesting to change name of MODE_LAST to MODE_POLL or are you suggesting to use MODE_PROGRESS?

Same for https://github.com/openucx/ucx/pull/6224#discussion_r573814460

Reply:
let's rename MODE_PROGRESS to MODE_POLL and use it
IMO, POLL is a clear and standard terminology

Reply:
Agreed. Will make the change now.